front-end/src/components/Post/Post.tsx

```diff
@@ -71,6 +72,17 @@ const Post = ( { className, data, isMotion = false, isProposal = false, isRefere
 	const [isEditing, setIsEditing] = useState(false);
 	const toggleEdit = () => setIsEditing(!isEditing);
 	const isOnchainPost = isMotion || isProposal || isReferendum || isTreasuryProposal;
+	const currentMeta = useContext(MetaContext);
+
+	useEffect(() => {
+		currentMeta.setMetaContextState((prevState) => {
+			return {
+				...prevState,
+				description: post?.content || prevState.description,
+				title: post?.title || prevState.title
+			};
+		});
+	}, [post, currentMeta]);
```

Comment:
This triggers an infinite loop I think, I've seen it in a referendum, in the console there's the maximum amount of update reached.

Reply:
done. Using title description and setcontext